lib/DidJwtStrategy.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const passport = require('passport');
+const util = require('util');
+const validate = require('bedrock-validation').validate;
+const BedrockError = bedrock.util.BedrockError;
+
+module.exports = Strategy;
+
+/**
+ * Creates a new didJwt Strategy for use with passport.
+ */
+function Strategy(options) {
+  const v = validate('authn-did-jwt.constructor', options);
+  if(!v.valid) {
+    throw v.error;
```

Comment:
While I like the idea of using the validation library here, I think it's a little weird and out of place. I think most people would be expecting to see `TypeError`s for misuse of the API. This also breaks with our code everywhere else that we handle JS API errors. I think we should change this to the usual style:

```js
options = options || {};
if(typeof options.foo !== 'string') {
  throw new TypeError('options.foo must be a string.');
}
// ...
```